packages/components/test/integration/TrezorLogo.test.js

```diff
@@ -0,0 +1,12 @@
+describe('TrezorLogo', () => {
+    beforeEach(() => {
+        cy.viewport(1024, 768);
+        cy.visit('/iframe.html?selectedKind=Other&selectedStory=TrezorLogo&full=0');
+    });
+
+    it(`TrezorLogo`, () => {
+        cy.getTestElement('trezor_logo')
```

Comment:
It would be best if we test both images, horizontal and also vertical.

Reply:
TrezorLogo.test.js will be removed after merge branch origin/components/enhancement/all-section-for-other
I did there the 'All' section for Other components and the tests are running from one file (as did for typography, buttons, forms, ...)